application/src/main/java/com/softserve/lv460/application/events/ResendTokenEvent.java

```diff
@@ -0,0 +1,20 @@
+package com.softserve.lv460.application.events;
+
+import lombok.Getter;
+import com.softserve.lv460.application.entity.ApplicationUser;
+import org.springframework.context.ApplicationEvent;
+
+@Getter
+@SuppressWarnings("serial")
+public class ResendTokenEvent extends ApplicationEvent {
+  private ApplicationUser user;
+  private String appUrl;
+  private String token;
+
+  public ResendTokenEvent(ApplicationUser user, String appUrl, String token) {
```

Comment:
use AllArgsConstructor

Reply:
I can't because I should call super(user) in constructor. This is not possible in Lombok.